.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: java
-dist: trusty
-install: "./etc/ci-setup.sh"
+dist: bionic
 script: ./mvnw clean install -Pit
-after_script: heroku keys:remove $USER@`hostname`
-
 matrix:
   fast_finish: true
   include:
-    - jdk: oraclejdk8
-    - jdk: oraclejdk11
+    - jdk: openjdk8
+    - before_install:
+        - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
```

Comment:
i wonder if you want to vendor, or fork this to prevent unexpected changes/errors. Although this is the one Travis recommends, so maybe it's safe.